packages/flutter/lib/src/material/material_button.dart

```diff
@@ -91,6 +92,9 @@ class MaterialButton extends StatelessWidget {
   /// If this is set to null, the button will be disabled.
   final VoidCallback onPressed;
 
+  /// Called when the button is long-pressed.
```

Comment:
I think it would be interesting to add in the comment that if it is null, the button will be disabled.

Reply:
But the button won't be disabled when long-press is null. There is no problem having a null long-press.

Reply:
Understood, sorry. The pull request really is very good.